TESTING.md

```diff
@@ -0,0 +1,15 @@
+# Testing
+Each BATS addon must be tested by its own [BATS test](https://github.com/garethahealy/bats-library/blob/master/test/tests.sh#L5-L12).
```

Comment:
probs a broken url here right?

Reply:
fixed to point to redhat-cop - but ye, its a deadlink until merged